akka-cluster/src/main/java/akka/cluster/protobuf/msg/ClusterMessages.java

```diff
@@ -15977,23 +15977,23 @@ public int hashCode() {
       }
 
       @java.lang.Override
-      public Builder newBuilderForType() { return newBuilder(); }
-      public static Builder newBuilder() {
+      public Version.Builder newBuilderForType() { return newBuilder(); }
+      public static Version.Builder newBuilder() {
```

Comment:
This is also odd, scala 3 complains of ambiguous reference, but this class is a generated one that we push. 

I fixed it by hand just to pass compilation, but such a solution won't last longer.

Reply:
That sounds like what @ignasi35 ran into for akka-persistence in #30406, https://github.com/lampepfl/dotty/issues/13106

Reply:
Yeah. It's the exact same cause: inner message on the `.proto` definition (https://github.com/akka/akka/blob/master/akka-cluster/src/main/protobuf/ClusterMessages.proto#L208-L209) causes ambiguous `Builder` reference.